optimade/models/jsonapi.py

```diff
@@ -1,81 +1,98 @@
-"""
-This module should reproduce https://jsonapi.org/schema
-"""
-from typing import Optional, Set, Union, Dict, Any
-from pydantic import BaseModel, UrlStr, constr, Schema
-
-
-class Meta(Dict[str, Any]):
+"""This module should reproduce JSON API v1.0 https://jsonapi.org/format/1.0/"""
+from typing import Optional, Set, Union
+from pydantic import BaseModel, UrlStr, Schema, validator
+
+
+__all__ = (
```

Comment:
Consider whether this specific model file should be left out of the `__init__`, i.e., not be importable from `optimade.models`.
The reason being, we want people to check against the derived models of these, not the original JSON API v1.0 models.